packages/core/src/services/documentLogs/create.ts

```diff
@@ -12,6 +12,7 @@ export type CreateDocumentLogProps = {
     resolvedContent: string
     customIdentifier?: string
     duration: number
+    createdAt?: Date
```

Comment:
why? should be automatically populated otherwise we have a problem

Reply:
This is only used for testing. I need to create old logs in a test environment.